Maintenance/511 requirement levels (#195)
ConsumerDataStandardsAustralia/standards-staging: slate/source/includes/nfrs/_session-requirements.md

## Session Requirements
A expiry time of a unique session should be set according to the statements included in the [Security Profile](#security-profile).

> After a unique session is expired it is expected that the Data Recipient Software Product, for the same customer, **MAY** establish a new session as long as the authorisation is still valid.

Reviewer: ```suggestion
After a unique session is expired it is expected that the Data Recipient Software Product, for the same Arrangement, **SHOULD** establish a new session as long as the authorisation is still valid.
```
